### PR TITLE
add battery contingency element

### PR DESCRIPTION
**Pull request by EtienneLt**
Signed-off-by: Etienne LESOT <etienne.lesot@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
#2303

### CODE REVIEW

File: contingency/contingency-api/src/main/java/com/powsybl/contingency/BatteryContingency.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/**
```
Comment (annetill) [inline contingency/contingency-api/src/main/java/com/powsybl/contingency/BatteryContingency.java:1]: `/**
 * Copyright (c) 2022, RTE (http://www.rte-france.com)
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 *  SPDX-License-Identifier: MPL-2.0
 */`

File: contingency/contingency-api/src/main/java/com/powsybl/contingency/Contingency.java [hunk 2]
```diff
@@ -247,6 +247,13 @@ public static ContingencyBuilder builder(String id) {
         return new ContingencyBuilder(id);
     }
 
+    /**
+     * Creates a new contingency on the branch whose id is given
```
Comment (annetill) [inline contingency/contingency-api/src/main/java/com/powsybl/contingency/Contingency.java:251]: Javadoc should fixed.

File: contingency/contingency-api/src/test/java/com/powsybl/contingency/tasks/BatteryTrippingTest.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
```
Comment (annetill) [inline contingency/contingency-api/src/test/java/com/powsybl/contingency/tasks/BatteryTrippingTest.java:5]: Can you add SPDX in the copyright.

File: contingency/contingency-api/src/main/java/com/powsybl/contingency/BatteryContingency.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+ * Copyright (c) 2022, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
```
Comment (annetill) [inline contingency/contingency-api/src/main/java/com/powsybl/contingency/BatteryContingency.java:2]: The copyright is still wrong :-) You are working with me at RTE, right?

**Review by annetill — APPROVED** [review]
Just 2 small fixes about copyright!

**Review by flo-dup — APPROVED** [review]

**Pull request merged**